### PR TITLE
[Merged by Bors] - vesting: add a sanity test that can import keys from disk

### PR DESCRIPTION
**Pull request by dshulyak**
it prints the following data, that can be used to sanity check that tool generated addresses that vm can work with.
the test itself spawn vesting and vault accounts. drains initial amount at vest start, and then incrementally drains whole amount until the end. ensuring amount that is added to the account makes sense.

to add another test case see examples in genvm/data.

```
vm_test.go:2478: vesting address: sm1qqqqqqp93fsceuhhcr5znke8xqdlmp69myxk32grx3mt9. parameters: required=1
        0 : 0x5c91eb16eb304e8b53ac310889fb0d4e60e8ccc90abf2e36d46c0efbd68aef75
        1 : 0xc1fd5dcd689f791022955cbc90483c13712b74634264ef6632b2f72f90411309
    vm_test.go:2479: vault address: sm1qqqqqqqzhehpu9ftpw8mr5k4tnkxprt8m7ptzzgkella3. parameters: owner = sm1qqqqqqp93fsceuhhcr5znke8xqdlmp69myxk32grx3mt9. total = 30000000000000 smidge. initial = 10000000000000 smidge. start = 105120. end = 420480
```

### CODE REVIEW

File: genvm/vm_test.go [hunk 1]
```diff
@@ -2416,6 +2420,151 @@ func FuzzParse(f *testing.F) {
...
+			drained := uint64(0)
+			remaining := uint64(meta.Total - meta.Initial)
+			fee := 0
+			// execute drain tx every 1000 layers
+			for i := constants.VestStart + 1; i < constants.VestEnd; i += 1000 {
+				before, err := vm.GetBalance(vestaddr)
+				require.NoError(t, err)
+
+				drain := uint64(meta.Total - meta.Initial)
+				drain *= (uint64(i) - constants.VestStart)
```
Comment (countvonzero) [inline genvm/vm_test.go]: this could overflow.

**Review by countvonzero — APPROVED** [review]

File: genvm/vm_test.go (hunk 1 above)
Comment (dshulyak) [reply genvm/vm_test.go]: thanks, that was one embarrasing mistake. i even convinced myself that it doesn't overflow by adding wrong test

**Comment by dshulyak** [comment]
bors merge

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**